### PR TITLE
Fixing a typo in Root Level Request Logger challenge description

### PR DESCRIPTION
**Pull request by LetMyPeopleCode**
Information is both singular and plural. The use of "informations" on line 9 was incorrect. Fixed.

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Closes #XXXXX

### CODE REVIEW

File: curriculum/challenges/english/05-apis-and-microservices/basic-node-and-express/implement-a-root-level-request-logger-middleware.english.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ challengeType: 2
...
+Before we introduced the <code>express.static()</code> middleware function. Now it’s time to see what middleware is, in more detail. Middleware functions are functions that take 3 arguments: the request object, the response object, and the next function in the application’s request-response cycle. These functions execute some code that can have side effects on the app, and usually add information to the request or response objects. They can also end the cycle sending the response, when some condition is met. If they don’t send the response, when they are done they start the execution of the next function in the stack. This is triggered calling the 3rd argument <code>next()</code>. More information in the <a href='http://expressjs.com/en/guide/using-middleware.html' target='_blank'>express documentation</a>.
```
Comment (thecodingaviator) [inline curriculum/challenges/english/05-apis-and-microservices/basic-node-and-express/implement-a-root-level-request-logger-middleware.english.md]:
```suggestion
Before, we had introduced the <code>express.static()</code> middleware function. Now it’s time to see what middleware is in more detail. Middleware functions are functions that take 3 arguments: the request object, the response object, and the next function in the application’s request-response cycle. These functions execute some code that can have side effects on the app, and usually add information to the request or response objects. They can also end the cycle by sending the response, when some condition is met. If they don’t send the response, when they are done they start the execution of the next function in the stack. This is triggered by calling the 3rd argument <code>next()</code>. More information in the <a href='http://expressjs.com/en/guide/using-middleware.html' target='_blank'>express documentation</a>.
```

Comment (RandellDawson) [reply curriculum/challenges/english/05-apis-and-microservices/basic-node-and-express/implement-a-root-level-request-logger-middleware.english.md]: I think `Before, we had introduced` should be `Earlier, we introduced`

Comment (YiddishNinja) [reply curriculum/challenges/english/05-apis-and-microservices/basic-node-and-express/implement-a-root-level-request-logger-middleware.english.md]: Agree with @RandellDawson. I had not thoroughly reviewed the text. Just tried to fix something that stood out. But now that @thecodingaviator surfaced it, I believe "Earlier" is clearer than "Before" in this context.

Who adds that change? I'm not fully up to speed on FCC Github etiquette.

Comment (RandellDawson) [reply curriculum/challenges/english/05-apis-and-microservices/basic-node-and-express/implement-a-root-level-request-logger-middleware.english.md]: @YiddishNinja It is your PR, so you would make the change.  But before you do, let's make sure @thecodingaviator is on board with the idea.

Comment (thecodingaviator) [reply curriculum/challenges/english/05-apis-and-microservices/basic-node-and-express/implement-a-root-level-request-logger-middleware.english.md]: I'm good with that too @RandellDawson . Sorry for the late reply.

Also Randell, I guess it's fine if @YiddishNinja makes the change

**Comment by RandellDawson** [comment]
@YiddishNinja I tried to resolve your merge conflict, but I assume you did not mark your PR as editable by maintainers.  Please check the box (see below), so we can resolve it on your behalf.  The option will be on the right side of the PR body below the Participants section.

image


Thank you.

**Comment by RandellDawson** [comment]
@YiddishNinja I tried to resolve your merge conflict, but I assume you did not mark your PR as editable by maintainers. In the future, please check the box (see below), so we can resolve it on your behalf. The option will be on the right side of the PR body below the Participants section.

This PR was holding up another PR making changes not to just this file, but many other files, so since we could not easily merge yours, we are closing this PR in favor of merging the other.

**Comment by YiddishNinja** [comment]
Updated with the allowing maintainers to edit. I'm in deep weeds on other stuff due to crazy bad luck and have to prioritize other things. So if the maintainers need to make changes to get this PR completed, please do.

**Pull request closed without merging**